pkg/controller/account/account_controller.go

```diff
@@ -636,22 +647,40 @@ func (r *ReconcileAccount) asyncRegionInit(reqLogger logr.Logger, currentAcctIns
 	// Initialize all supported regions by creating and terminating an instance in each
 	r.InitializeSupportedRegions(reqLogger, currentAcctInstance, awsv1alpha1.CoveredRegions, creds)
 
+	var err error = nil
```

Comment:
This is an ineffectual assignment. Go already initialises this to `nil`.
```suggestion
	var err error
```